### ISSUE TITLE
googleAuth_jsUI replacement

### ISSUE THREAD

**Issue by didllicek** [MarkEdmondson1234/googleAuthR#217]
Hi,

more than an issue report I would like to ask if there will be (or maybe already is, but I just can not see it) any replacement for googleAuth_jsUI in version 2.0.0. We used this function for authentification at our R&D webpage https://rnd.logio.cz/ and I don't see any way how could I do something like this in new version.

Thank you for your advice :)

**Comment by MarkEdmondson1234** [comment]
There isn't any planned but if you need it I can put it back in for 2.1. But can you pin your version at 1.4.1? That way it will always work as you expected.

**Comment by MarkEdmondson1234** [comment]
I didn't include it as more modern alternatives like firebase (R package) exist now that may be preferable.

**Comment by MarkEdmondson1234** [comment]
https://firebase.john-coene.com/

**Comment by MarkEdmondson1234** [comment]
The current API is also going to be sunset, with advice to move to https://developers.google.com/identity/oauth2/web

**Comment by didllicek** [comment]
Thank you for the warning. I got some alert mail also from google. Do I understand correctly that after one year it will not work as expected anymore? :( 
I tried to change googleAuthR library for Firebase, but the application is registered in the Google developer console and I need restricted access only for people within the organization, which I really do not know how to do with Firebase. Do you think it's possible to use Firebase like this? or any other package? 
I'm a very junior programmer and it was quite difficult for me to find a working solution thanks to your library, so I'm a little bit worried. :(

**Comment by MarkEdmondson1234** [comment]
Yes I got the same email, but I think it will be a small change to update the JS library to the new one - its happened at least once in the past before. So long as I (or someone via a pull request) updates it before 2023 then it should be ok.  I have noted it in this issue to make sure its covered in the next release.

**Comment by didllicek** [comment]
Thank you very much! :) I'll let you know if I figure out meantime how to change for Firebase or some other library.

**Comment by Askarolla** [comment]
Hello @MarkEdmondson1234 , 

I am the main developer of our company [R&D webpage](https://rnd.logio.cz/) which my colleague inquired about with respect to the googleAuth_jsUI in your library. As she mentioned, your library provides a combination of unique functionalities, and we would therefore like to keep using it in our implementation.

I am writing to ask if you are still planning to include the function in the new library version before 03/2023 when, according to the notifications we keep getting, it would stop working?

Thank you so much :)
